mix.exs

```diff
@@ -2,7 +2,7 @@ defmodule UeberauthCognito.MixProject do
   use Mix.Project
 
   @source_url "https://github.com/mbta/ueberauth_cognito"
-  @version "0.3.0"
+  @version "0.3.1"
```

Comment:
question(non-blocking): is it worth making this `1.0.0`?

Reply:
Great question! `ueberauth` itself is not at `1.0.0` yet so that gives me pause. But might still be worth doing. I'll make a tracking ticket to think about and discuss this.